src/views/Dashboard.vue

```diff
@@ -182,37 +141,25 @@ export default {
       return list
     }
   },
-  mounted () {
+  mounted: function () {
     this.loadData()
+    // This is required to re-draw the arrows, sorry :P
+    let that = this
+    this.$nextTick(function() {
+      window.addEventListener("resize", function () {that.$forceUpdate()})
+    }(that))
+  },
```

Comment:
I think with fancy modern javascript, you can use the arrow syntax which will preserve the "this"... like so:

```
this.$nextTick(() => window.addEventListener("resize", () => this.$forceUpdate)))
```

Reply:
Actually I tested this (and other alternatives) and couldn't make it work :man_shrugging: